go-controller/pkg/metrics/master.go

```diff
@@ -189,6 +189,43 @@ var metricEgressIPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of defined egress IP addresses",
 })
 
+var metricEgressIPAssignLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "egress_ips_assign_latency_seconds",
+	Help:      "The latency of egress ip assignment to ovn nb database",
+	Buckets:   prometheus.ExponentialBuckets(.001, 2, 15),
+})
+
+var metricEgressIPUnassignLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "egress_ips_unassign_latency_seconds",
+	Help:      "The latency of egress ip unassignment from ovn nb database",
+	Buckets:   prometheus.ExponentialBuckets(.001, 2, 15),
+})
+
+var metricEgressIPNodeReacheableCount = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
why do we care if its reachable? Should we only care if its not reachable?

Reply:
We already have a metric where we know the number of assigned egress IPs so we have an idea already how many egress IPs are assigned. Do we need to know how many successful reach-ability attempts there were?

Reply:
good point. It may be overkill, so let's get rid of it and just keep metricEgressIPNodeUnreacheableCount